app/uk/gov/hmrc/bindingtariffclassification/model/search/Search.scala

```diff
@@ -32,7 +34,7 @@ case class Filter
 (
   queueId: Option[String] = None,
   assigneeId: Option[String] = None,
-  status: Option[String] = None,
+  status: Option[Set[CaseStatus]] = None,
```

Comment:
**status** make sense to me, but maybe now a plural **statuses** would be more self-explanatory.